infrastructure/storage-provisioning/README.md

```diff
@@ -85,3 +85,25 @@ After that you can delete test with commands
 $ kubectl delete -f examples/wordpress.yaml
 $ kubectl delete -f examples/mysql.yaml
 ```
+
+## Rook NFS
+To use the user's personal storage we decided to use the NFS server provided by Rook-Ceph ([here](https://rook.io/docs/rook/v1.10/Storage-Configuration/NFS/nfs/)).
+
+With the `CephNFS` CRD we configure a NFS-Ganesha server to provide the access to the user's personal storage via NFS inside the cluster.
+
+A `StorageClass` is used for the dynamic provisioning of the user's storage. In this `StorageClass` is specified, via specific parameters for the Rook Operator, to create also a NFS share to which the user can later connect to via the NFS protocol.
```

Comment:
The user's storage is dynamically provisioned through a `StorageClass` that contains a set of parameters that tell the Rook Operator to create also a NFS share; this is later mounted from user's VMs and containers though the NFS protocol.